DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,192 @@
+﻿@inject IConfiguration configuration
+@using DigitalLearningSolutions.Web.Extensions
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using Microsoft.Extensions.Configuration
+@model EditCourseDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = "Edit course details";
+  ViewData["Application"] = DlsSubApplication.TrackingSystem.Name;
+  ViewData["HeaderPath"] = DlsSubApplication.TrackingSystem.HeaderPath;
+  ViewData["HeaderPathName"] = DlsSubApplication.TrackingSystem.HeaderPathName;
+  var cancelLinkData = Html.GetRouteValues();
+  var customisationId = Html.GetRouteValueByKey("customisationId");

```

Comment:
Stella once told me that it's messy to have the customisationId in the route parameter and on the model because there's a possibility they could differ. But I this is probably messy too so I changed it back to have the customisationId on the model, and added a check in the controller that they are the same.

Reply:
That's potentially a thing to split out into ViewModel/FormData then. I don't really see why there would be a situation where they differ unless there is an error when implementing it.

Reply:
Cool, I've split it into a View Model and Form Data, let me know how that looks :)